cabal.project

```diff
@@ -348,11 +90,21 @@ package snap-server
 package comonad
   flags: -test-doctests
 
-package cardano-ledger-alonzo-test
-  tests: False
-
 allow-newer:
   *:aeson,
   monoidal-containers:aeson,
   size-based:template-haskell,
   snap-server:attoparsec,
+
+-- Open PR upstream, maintainer unresponsive, hopefully short-lived fork.
```

Comment:
Might be nice to have documentation saying when downstream repos need these overrides as well.

Reply:
Oh dear yes.

Reply:
Happy to add this, just not sure where. Probably this should be in the README. I'll add a section about "using the Node as a Haskell dependency".

Reply:
Could we get that info on haskell package level? i.e. likely these won't be needed if we only use `cardano-api`?

Reply:
I don't know the answer, unfortunately.